FloconDesktop/composeApp/src/desktopMain/kotlin/io/github/openflocon/flocondesktop/about/Contributor.kt

```diff
@@ -22,5 +22,47 @@ internal val contributors = listOf(
         lastName = "Teyssandier",
         profile = "https://github.com/doTTTTT",
         image = "https://avatars.githubusercontent.com/u/13266870?v=4"
+    ),
+    Contributor(
+        firstName = "Wouter",
+        lastName = "van der Velde",
+        profile = "https://github.com/Woutervdvelde",
+        image = "https://github.com/Woutervdvelde.png"
+    ),
+    Contributor(
+        firstName = "Gustavo",
+        lastName = "de Santos Garcia",
+        profile = "https://github.com/gdesantos",
+        image = "https://github.com/gdesantos.png"
+    ),
+    Contributor(
+        firstName = "Quentin",
+        lastName = "HUET",
+        profile = "https://github.com/Karambar",
+        image = "https://github.com/Karambar.png"
+    ),
+    Contributor(
+        firstName = "Sebastian",
+        lastName = "Neubauer",
+        profile = "https://github.com/Flakebi",
+        image = "https://github.com/Flakebi.png"
```

Comment:
@florent37 
Hi, I appreciate the shout out, but the links are wrong in my case :)
My account is: `https://github.com/snappdevelopment`
Also below in the html 👇